src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -176,9 +177,15 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef(
       splitPanelPosition: splitPanelPreferences?.position,
     });
 
+    const discoveredBreadcrumbs = useGetGlobalBreadcrumbs();
+
     const hasToolbar =
       !embeddedViewMode &&
-      (!!resolvedNavigation || !!breadcrumbs || splitPanelToggleConfig.displayed || drawers!.length > 0);
+      (!!resolvedNavigation ||
+        !!breadcrumbs ||
+        !!discoveredBreadcrumbs ||
```

Comment:
I think we don't need the double exclamation marks here, and they make the code difficult to read. We could make this block more readable by removing them, and converting only the result to a boolean:
```
const hasToolbar = Boolean(
      !embeddedViewMode &&
      (resolvedNavigation ||
        breadcrumbs ||
        discoveredBreadcrumbs ||
        splitPanelToggleConfig.displayed ||
        drawers!.length > 0)
);
```

Reply:
Applied